samples/Microsoft.AspNet.SignalR.Samples/Scripts/hubs.js

```diff
@@ -1,720 +0,0 @@
-/*!
```

Comment:
Does this break the sample?

Reply:
No. The file was already auto generated at build time. I deleted it and added it to .gitignore because I didn't want to have to update this file every time the version changes.